examples/without_gui_qt.py

```diff
@@ -0,0 +1,10 @@
+"""
+Alternative to using napari.gui_qt() context manager. 
+"""
+
+from skimage import data
+import napari
+
+viewer = napari.view_image(data.astronaut(), rgb=True)
+
+napari.run_app()
```

Comment:
```suggestion
viewer = napari.view_image(data.astronaut(), rgb=True)
# You can also take screenshots before running the app
screenshot = viewer.screenshot()

print('Maximum value', screenshot.max())

# Run the app to see the napari viewer. If you only wanted the screenshot
# then you could skip this entirely
napari.run_app()
```

No need to accept this suggestion, but this is amazing!! This is exactly what I wanted from #2029 and will be very useful for automatically making movies etc as you never need to call `run_app`